packages/security/src/enums.ts

```diff
@@ -27,57 +27,147 @@ export const chainIdToConfig: { [chainId: number]: ChainConfig } = {
+    },
+  },
+  liquidity: {
+    HIGH: {
+      title: "High trade liquidity",
+      explanation:
+        "How liquid an asset is affects how risky it is to hold it. Liquid assets are traded in many places and with good volume. The asset held by this vault has high liquidity. This means that you can exchange your earnings easily in plenty of places.",
+    },
+    LOW: {
+      title: "Low trade liquidity",
+      explanation:
+        "How liquid an asset is affects how risky it is to hold it. Liquid assets are traded in many places and with good volume. The asset held by this vault has low liquidity. This means that it isn't as easy to swap and you might incur high slippage when doing so.",
+    },
+  },
+  mktCap: {
+    LARGE: {
+      title: "High market cap, low volatility asset",
+      explanation:
+        "The market capitalization of the crypto asset directly affects how risky it is to hold it. Usually a small market cap implies high volatility and low liquidity. The asset held by this vault has a large market cap. This means it's potentially a highly safe asset to hold. The asset has a high potential to stick around and grow over time.",
+    },
+    MEDIUM: {
+      title: "Medium market cap, medium volatility asset",
+      explanation:
+        "The market capitalization of the crypto asset directly affects how risky it is to hold it. Usually a small market cap implies high volatility and low liquidity. The asset held by this vault has a medium market cap. This means it's potentially a safe asset to hold. The asset has potential to stick around and grow over time.",
+    },
+    SMALL: {
+      title: "Small market cap, high volatility asset",
+      explanation:
+        "The market capitalization of the crypto asset directly affects how risky it is to hold it. Usually a small market cap implies high volatility and low liquidity. The asset held by this vault has a small market cap. This means it's potentially a risky asset to hold. The asset has low potential to stick around and grow over time.",
+    },
+    MICRO: {
+      title: "Micro market cap, Extreme volatility asset",
+      explanation:
+        "The market capitalization of the crypto asset directly affects how risky it is to hold it. Usually a small market cap implies high volatility and low liquidity. The asset held by this vault has a micro market cap. This means it's potentially a highly risky asset to hold. The asset has low potential to stick around.",
+    },
+  },
+  supplyCentralised: {
+    CENTRALIZED: {
+      title: "Few very powerful whales",
+      explanation:
+        "When the supply is concentrated in a few hands, they can greatly affect the price by selling. Whales can manipulate the price of the coin. The more people that have a vested interest over a coin, the better and more organic the price action is.",
+    },
+    DECENTRALIZED: {
+      title: "Decentralized",
+      explanation: "Decentralized",
```

Comment:
"The supply is concentrated is spread across many users instead of being concentrated in the hands of a few whales, which makes it so that price manipulations are less likely to occur. The more people that have a vested interest over a coin, the better and more organic the price action is.",

Reply:
Added!